atlas/src/shared/components/VideoPreview/VideoPreview.styles.tsx

```diff
@@ -91,22 +92,36 @@ export const StyledAvatar = styled(Avatar)<ChannelProps>`
   cursor: ${({ channelClickable }) => (channelClickable ? 'pointer' : 'auto')};
 `
 
-export const TitleHeader = styled.h3<MainProps & ScalesWithCoverProps>`
+const fadeIn = keyframes`
```

Comment:
Couldn't we add that animation to the `InfoContainer` in the `VideoPreviewBase.style.tsx`? Then we can fade in the whole container, not the individual nodes

Reply:
That would be better indeed, but doing so changes the color of the placeholders inside `InfoContainer` when loading, so I suggest we keep this as is for now

Reply:
Okay, we can skip that, although there is another problem. Currently the fade in animations aren't in sync, i.e. this uses different duration and easing than the cover. I suggest we define the keyframes and the animation in the base styles file and import it here so these parameters are always the same. Like so:

```js
const fadeIn = keyfreames...

export const fadeInAnimation = css`
  animation: ${fadeIn} 0.5s ease-in;
`
```